internal/workflow/processing.go

```diff
@@ -439,6 +440,16 @@ func (w *ProcessingWorkflow) SessionHandler(
 		return err
 	}
 
+	// Validate PREMIS.
+	activityOpts := withActivityOptsForRequest(sessCtx)
+	err := temporalsdk_workflow.ExecuteActivity(activityOpts, xmlvalidate.Name, xmlvalidate.Params{
+		XMLPath: filepath.Join(tinfo.TempPath, "data", "metadata", "premis.xml"),
+		XSDPath: "/home/enduro/premis.xsd",
```

Comment:
This should be configurable:

```
[validatePremis]
xsdPath = "/home/enduro/premis.xsd"
```

Or something like that. It will require changes in the config package.

Reply:
As noted in Slack, it should also be optional, so maybe:

```
[validatePremis]
enabled = true
xsdPath = "/home/enduro/premis.xsd"
```

It could also be checked on the `xsdPath` var directly if you don't want to add an extra var. If `xsdPath` is empty, validation is disabled.

Reply:
Thinking about possible errors from the activity, we should check the existence of the XSD file when reading the config. We don't want to let the workers start knowing all workflows will fail in that activity.

Right now, we use the same config file for all services/containers, so this check will fail for the `enduro` container, as you are only copying the XSD file in the `enduro-am-worker` and the `enduro-a3m-worker` containers. You should be able to solve that issue changing that config value in the environment:

https://github.com/artefactual-sdps/enduro/blob/main/hack/kube/base/enduro.yaml#L30

Adding a new env. var., based on the example:

```
- name: ENDURO_VALIDATE_PREMIS_ENABLED
  value: "false"
```

If the config validation checks for `Enabled` first before validating the existence of the XSD file, then you don't need to overwrite the path from the TOML, just disable it.

Similar, if you decide to use a single var:

```
- name: ENDURO_VALIDATE_PREMIS_XSD_PATH
  value: ""
```

If the config validation sees an empty value it doesn't check of its existence.